src/components/modules/Authentcation/Login/Login.tsx

```diff
@@ -1,5 +1,153 @@
+            <div
+              className="bg-authchoosuserbackgroundcolor w-48 text-center py-5 rounded-xl transition-all cursor-pointer hover:outline outline-auttextcolor"
+              onClick={navigateRegister}
+            >
+              {/* <Link to="/register" className="text-6xl"> */}
+              <i className="fa-solid fa-user-plus text-6xl"></i>
+              {/* </Link> */}
+              <h6 className="font-semibold pt-2">sign up</h6>
+            </div>
+          </div>
+          <form onSubmit={handleSubmit(onsubmit)}>
+            <label htmlFor="email" className="block">
+              Registered email address
+            </label>
+            <div className="flex items-center border-2 border-gray-600 rounded-3xl px-3 py-2 focus-within:border-auttextcolor2">
+              <i className="fa-solid fa-envelope bg-authbackground text-xl mr-3"></i>
+
+              <input
+                type="email"
+                placeholder="Type your email"
+                className="  w-full bg-authbackground border-none outline-none"
+                {...register("email", {
+                  required: "email is require",
+                  pattern: {
+                    value: VALIDATIONS.emailRegex,
+                    message: "Invalid email",
+                  },
+                })}
+              />
+            </div>
+            {errors.email ? <span>{errors.email.message}</span> : ""}
+
+            <label htmlFor="password" className="block">
+              password
+            </label>
+            <div className="flex items-center border-2 border-gray-600 rounded-3xl px-3 py-2 focus-within:border-auttextcolor2">
+              <i className="fa-solid fa-key bg-authbackground text-xl mr-3"></i>
+
+              <input
+                type="password"
+                placeholder="password"
+                className="  w-full bg-authbackground border-none outline-none"
+                {...register("password", { required: "password is required" })}
+              />
+            </div>
+            {errors.password ? <span>{errors.password.message}</span> : ""}
+            <div className="flex justify-between items-center py-3">
+              {!isloading ? (
+                <button
+                  className="py-3 px-10 rounded-2xl bg-white text-black font-semibold  hover:bg-lime-500 transition-all "
+                  type="submit"
+                >
+                  sign in <i className="fa-regular fa-circle-check"></i>{" "}
+                </button>
+              ) : (
+                <button
+                  disabled
+                  type="button"
+                  className=" py-2.5 px-5 me-2 text-sm font-medium text-gray-900 bg-white rounded-lg border border-gray-200 hover:bg-gray-100 hover:text-blue-700 focus:z-10 focus:ring-2 focus:ring-blue-700 focus:text-blue-700 dark:bg-gray-800 dark:text-gray-400 dark:border-gray-600 dark:hover:text-white dark:hover:bg-gray-700 inline-flex items-center"

```

Comment:
I suggest using tailwind prettier plugin, it will format the classes nicely through the application. You can find the link here:
https://tailwindcss.com/blog/automatic-class-sorting-with-prettier